modular_skyrat/code/game/objects/items/misc.dm

```diff
@@ -0,0 +1,79 @@
+//hazard vest armor
+/obj/item/clothing/suit/armor/hazard
+	name = "armored hazard vest"
+	desc = "A hazard vest with plasteel and metal plates taped on it. It offers minor protection against kinetic damage, but slows you down a significant bit."
+	icon = 'modular_skyrat/icons/obj/clothing/armor.dmi'
+	alternate_worn_icon = 'modular_skyrat/icons/mob/suit.dmi'
+	alternate_worn_icon_digi = 'modular_skyrat/icons/mob/suit_digi.dmi'
+	icon_state = "makeshiftarmor"
+	item_state = "makeshiftarmor"
+	w_class = 3
+	blood_overlay_type = "armor"
+	armor = list("melee" = 20, "bullet" = 10, "laser" = 5, "energy" = 5, "bomb" = 10, "bio" = 10, "rad" = 35, "fire" = 10, "acid" = 25)
+	slowdown = 0.35 //not completely insignificant but will not help you either
+
+//trayshield
+/obj/item/shield/trayshield
+	name = "tray shield"
+	desc = "A makeshift shield that won't last for long."
+	icon = 'modular_skyrat/icons/obj/weapons.dmi'
+	lefthand_file = 'modular_skyrat/icons/mob/inhands/lefthand.dmi'
+	righthand_file = 'modular_skyrat/icons/mob/inhands/righthand.dmi'
+	icon_state = "trayshield"
+	force = 5
+	throwforce = 5
+	throw_speed = 2
+	throw_range = 3
+	w_class = WEIGHT_CLASS_BULKY
+	attack_verb = list("shoved", "bashed")
+	block_chance = 40
+	var/damage_received = 0 //Amount of damage the shield has received
+	var/max_damage = 100 //Amount of max damage the trayshield can withstand
+
```

Comment:
Doesn't the wooden buckler have 25 block chance?
The materials required to craft are fairly common. The wooden buckler is more expensive, so this should either be brought in line with it.

Reply:
You're kind of right, but it has 30 block chance. I'll make this one have 25 block chance.

Reply:
Actually it'll be 30, since it has the disadvantage of breaking very easily